diff --git a/node_modules/@material/material-color-utilities/index.d.ts b/node_modules/@material/material-color-utilities/index.d.ts
index 799ace3..aefc6d2 100644
--- a/node_modules/@material/material-color-utilities/index.d.ts
+++ b/node_modules/@material/material-color-utilities/index.d.ts
@@ -30,7 +30,6 @@ export * from './quantize/quantizer_map.js';
 export * from './quantize/quantizer_wsmeans.js';
 export * from './quantize/quantizer_wu.js';
 export * from './scheme/scheme.js';
-export * from './scheme/scheme_android.js';
 export * from './scheme/scheme_content.js';
 export * from './scheme/scheme_expressive.js';
 export * from './scheme/scheme_fidelity.js';
diff --git a/node_modules/@material/material-color-utilities/index.js b/node_modules/@material/material-color-utilities/index.js
index d10e9ec..66a1fc8 100644
--- a/node_modules/@material/material-color-utilities/index.js
+++ b/node_modules/@material/material-color-utilities/index.js
@@ -30,7 +30,6 @@ export * from './quantize/quantizer_map.js';
 export * from './quantize/quantizer_wsmeans.js';
 export * from './quantize/quantizer_wu.js';
 export * from './scheme/scheme.js';
-export * from './scheme/scheme_android.js';
 export * from './scheme/scheme_content.js';
 export * from './scheme/scheme_expressive.js';
 export * from './scheme/scheme_fidelity.js';
